exporter/zipkin/zipkin.go

```diff
@@ -50,6 +54,36 @@ func NewExporter(reporter reporter.Reporter, localEndpoint *model.Endpoint) *Exp
 	}
 }
 
+type Options struct {
+	// EndpointHostPort is the host:port combination for the Zipkin service
+	// e.g. 192.168.0.2:8080
+	EndpointHostPort string
```

Comment:
What about using `net.IPV4` instead?

Reply:
Thanks @jcchavezs for the review and welcome to OpenCensus! I mention this because this is the first time that we are interacting :)

So I think net.IPV4 might be too restrictive in the type:
localhost:
localhost:8080
foo.bar:80
etc are all valid combinations for a host:port combination but wouldn't be a net.IPV4.

Reply:
Right, sorry I missed the `port`.